template.typ

```diff
@@ -51,6 +51,14 @@
         #set text(8pt)
         979-8-3503-2934-6/23/\$31.00 ©2023 IEEE
       ]
+      else [
```

Comment:
Arguably we changed it from the IEEE-style, so the copyright could be removed. Then to get the same effect, just replace the whole `footer` block at L:47 with

```suggestion
numbering: "1 / 1",
```